redis.conf

```diff
@@ -1088,6 +1088,24 @@ oom-score-adj no
 
 oom-score-adj-values 0 200 800
 
+
+#################### KERNEL transparent hugepage CONTROL ######################
+
+# Since Linux 3.15, it is possible to disable THP(transparent hugepage) setting
+# for a specified process.
+#
+# Transparent Hugepage Support is an alternative means of using huge pages for
+# the backing of virtual memory with huge pages that supports the automatic
+# promotion and demotion of page sizes and without the shortcomings of hugetlbfs.
+# But fixing a Copy-On-Write page fault for THP takes more time, and latency
+# jitter may occurs after forking a child process.
+#
+# For general purpose, the recommended config is 'yes', redis will try to disable
+# THP for itself when startup. And if you have a specific use of THP, set this
+# to 'no'.
+
+disable-thp yes
```

Comment:
```suggestion
#################### LINUX KERNEL THP CONTROL #################################

# Usually the kernel Transparent Huge Pages control is set to "madvise" or
# or "never" by default (/sys/kernel/mm/transparent_hugepage/enabled), in which
# case this config has no effect. On systems in which it is set to "always",
# redis will attempt to disable it specifically for the redis process in order
# to avoid latency problems specifically with fork(2) and CoW.
# If for some reason you prefer to keep it enabled, you can set this config to
# "no" and the kernel global to "always".
#
# disable-thp yes
```